Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundMainViewModel.swift

```diff
@@ -69,9 +69,9 @@ final class PlaygroundMainViewModel: ObservableObject {
         }
     }
 
-    @Published var enableTransactionsPermission: Bool = PlaygroundUserDefaults.enableTransactionsPermission {
+    @Published var enableOwnershipPermission: Bool = PlaygroundUserDefaults.enableOwnershipPermission {
```

Comment:
Bunch of this code under `Example/FinancialConnections Example/` is just testing/debug code that doesn't need any deep review